get_imports.py

```diff
@@ -1,35 +1,38 @@
-import requests
+# SPDX-FileCopyrightText: 2021 foamyguy
+# SPDX-FileCopyrightText: 2019 Nicholas Tollervey, written for Adafruit Industries
+#
+# SPDX-License-Identifier: MIT
+
+"""
+Get the list of required libraries based on a file's imports
+"""
+
 import json
 import os
 import findimports
+import requests
 
 BUNDLE_DATA = "latest_bundle_data.json"
 BUNDLE_TAG = "latest_bundle_tag.json"
 
-LEARN_GUIDE_REPO = os.environ.get('LEARN_GUIDE_REPO', "../../Adafruit_Learning_System_Guides/")
+LEARN_GUIDE_REPO = os.environ.get(
+    "LEARN_GUIDE_REPO", "../../Adafruit_Learning_System_Guides/"
```

Comment:
With the scrips in the root of the repo this is now one level off in my environment, so `../Adafruit_Learning_System_Guides/` is the value I use now. 

But this is dependent on the environment it's running in and may end up needing to change again when it's setup with the CI, I'm going to merge it for now. Perhaps when running in CI there is some other way it can access the repo that it's running inside of

Reply:
Ah, actually I just noticed this is an environment variable now and the string given I guess is the default used if it's not present. So the hardcoded value doesn't need to change on different environments. 

Thanks for all of these  improvements!